Makefile

```diff
@@ -14,12 +14,13 @@ SWAGGER := $(GOPATH)/bin/swagger
 GOBINDATA := $(GOPATH)/bin/go-bindata
 
 VERSION_PACKAGE := github.com/vmware/dispatch/pkg/version
+CLICMD_PACKAGE := github.com/vmware/dispatch/pkg/dispatchcli/cmd
 
-GO_LDFLAGS :="
-GO_LDFLAGS += -X $(VERSION_PACKAGE).version=$(VERSION)
+GO_LDFLAGS := -X $(VERSION_PACKAGE).version=$(VERSION)
 GO_LDFLAGS += -X $(VERSION_PACKAGE).buildDate=$(shell date +'%Y-%m-%dT%H:%M:%SZ')
 GO_LDFLAGS += -X $(VERSION_PACKAGE).commit=$(shell git rev-parse HEAD)
-GO_LDFLAGS +="
+
+CLI_LDFLAGS := -X $(CLICMD_PACKAGE).imagesB64=$(shell cat images.yaml | gzip | base64)
```

Comment:
NIT: could be `gzip -c images.yaml | base64`, one cat's life saved ;)

Reply:
No! This would kill the cat! 🙀